.gitignore

```diff
@@ -0,0 +1,9 @@
+tmp/
+temp/
+
+# Ignore all pickle files
+*.pickle*
+*.pkl*
+
+# Ignore all data files
+*/data/*
```

Comment:
Maybe a `./**/data` or something similar would help? Also, `mlruns` and `results` directories pop up alot.